src/codeu/chat/DerbyStore.java

```diff
@@ -137,7 +204,7 @@ public void updateLastMessage(Message m) throws SQLException {
 				// Add each created user to the store object
 				allUsers.insert(userid, user);
 			}
-			System.out.println("Successfully retrieved users");
+			LOG.info("Accessed users table.");
 		}
 		catch (Exception ex) {
 			ex.printStackTrace();
```

Comment:
One minor note: if this is an expected Exception, then you might not want to print its stack trace. Mostly this is to avoid spamming exceptions to the console when you run the tests, since that makes it harder to see if something failed. Maybe a simpler log statement would be better?